<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditButtonDialog</class>
 <widget class="QDialog" name="EditButtonDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Button</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="labelInput">
       <item>
        <widget class="QLabel" name="labelLabel">
         <property name="text">
          <string>Label:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="labelEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="fileInput">
       <item>
        <widget class="QLabel" name="fileLabel">
         <property name="text">
          <string>File:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="fileEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="fileButton">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>26</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Select</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionOpenFileDialog">
   <property name="text">
    <string>OpenFileDialog</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditButtonDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditButtonDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileButton</sender>
   <signal>clicked()</signal>
   <receiver>EditButtonDialog</receiver>
   <slot>fileDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>333</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>377</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>labelEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>EditButtonDialog</receiver>
   <slot>updateLabelText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>EditButtonDialog</receiver>
   <slot>updateFileText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>fileDialog()</slot>
  <slot>updateFileText(QString)</slot>
  <slot>updateLabelText(QString)</slot>
 </slots>
</ui>
